components/Modal/Modal.Styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const ModalBackround = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1001;
```

Comment:
In general, it's a good practice to keep `z-index values` as low and as close to each other as possible. But that's more of an advice.